SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -155,12 +155,32 @@ SimpleNumber : Number {
 		var diff = round - this;
 		if (abs(diff) < tolerance) {
 			^this + (strength * diff)
-		}{
+		} {
+			^this
+		}
+	}
+
+	snap { arg resolution = 1.0, margin = 0.05, strength = 1.0;
+		var round = round(this, resolution);
+		var diff = round - this;
+		if ( abs(diff) < margin ) {
```

Comment:
why are there suddenly spaces in the parens now? also FWIW i think our semiformal style is to not add a space after `if` since it is also a method and not a true control structure as in many other languages:

`if(abs(diff) < margin)`

Reply:
I have no idea how those spaces got there. Wasn't deliberate.